### DISCUSSION TITLE
Welcome to Catalyst-Alliance-Co-ordination

### DISCUSSION THREAD

**Discussion by stephen-rowan** [The-Catalyst-Alliance/Catalyst-Alliance-Co-ordination#1] [Announcements]
## Catalyst-Alliance-Co-ordination

Welcome to "Catalyst-Alliance-Co-ordination" this is the GitHub repository where "The-Catalyst-Alliance" GitHub Organisation will be managed by its members, maintainers, contributors.

At present these are the active Project Boards - (https://github.com/The-Catalyst-Alliance/Catalyst-Alliance-Co-ordination/projects)

## Catalyst-Alliance-Co-ordination - GitHub Set-Up - Project Board
This is where issues related to the set-up and development of the GitHub Organization are managed.

https://github.com/The-Catalyst-Alliance/Catalyst-Alliance-Co-ordination/projects/1

- Here is an example of an Issue raised on the "Catalyst-Alliance-Co-ordination - GitHub Set-Up" project board - [The-Catalyst-Alliance GitHub Organization Set-Up](https://github.com/The-Catalyst-Alliance/Catalyst-Alliance-Co-ordination/issues/2)

## The-Catalyst-Alliance GitBook  - Project Board
This is where issues related to the set-up and development of the [The-Catalyst-Alliance GitBook](https://catalyst-swarm.gitbook.io/the-catalyst-alliance/) are managed.

https://github.com/The-Catalyst-Alliance/Catalyst-Alliance-Co-ordination/projects/2

- Here is an example of an Issue raised on the "The-Catalyst-Alliance GitBook" project board - [The-Catalyst-Alliance GitBook Scaleability](https://github.com/The-Catalyst-Alliance/Catalyst-Alliance-Co-ordination/issues/3)

**Comment by newmindflow** [comment]
Nice to see this up and running @stephen-rowan . It should serve as a good starting point.

**Comment by newmindflow** [comment]
@stephen-rowan Do you think forking the Council of DAOs discussions here would make sense?
This would probably be a good place particularly for the HUB initiative.

**Reply by stephen-rowan** [reply]
Unfortunately forking a repository to another GitHub Organisation leaves its issues and discussions behind, which is where most of the content is for https://github.com/Council-of-DAOs/Council-Hub . I could create a small archive of the Council-Hub in GitBook format on The-Catalyst-Alliance GitHub if the TCA agree ?
